vuepress/docs/next/tutorials/solution/pda-tutorial.md

```diff
@@ -60,9 +60,12 @@ Follow the steps below to configure service permissions and connections.
 
 1. [Login to your Keycloak instance](../../docs/consume/identity-management.md#logging-into-your-keycloak-instance) as an admin.
 
-2. [Assign client roles](../../docs/consume/identity-management.md#authorization) to authorize permissions.
-- Select `pn-b0ceabd7-efbd66b6-entando-pda-plugin-server` from `Client Roles`.
-- Select all roles from `Available Roles`.
+2. Set up permissions to configure the service: 
+   - Give at least one user the ability to manage the plugin by granting the `pda-admin` role. Assign the `pda-admin` role for the `pn-efbd66b6-b0ceabd7-entando-pda-plugin-server` client. Go to [Assign user and client roles](../../docs/consume/identity-management.md#authorization) for more details.
```

Comment:
@nshaw I made this up since I could not check. Don't even know if it's necessary.

Reply:
@jyunmitch the original language from Lydia was correct, "select all roles...".

Reply:
same comment wrt "Assign user..."